### PR TITLE
Add 2 packages to existing container

### PR DESCRIPTION
**Pull request by LaurenceKuhl**

### CODE REVIEW

File: combinations/hash.tsv [hunk 1]
```diff
@@ -603,4 +603,4 @@ lumpy-sv=0.3.1	quay.io/bioconda/base-glibc-busybox-bash:3.1	3
 bwameth=0.2.7,bwa-mem2=2.2.1
 pasta=1.9.0,pigz=2.8
 longphase=1.7.3,htslib=1.20
-r-base,r-igraph,r-dplyr
+r-base,r-igraph,r-dplyr,r-tidyr,r-readr,r-ggplot2
```
Comment (bgruening) [inline combinations/hash.tsv:606]: Please add versions

Comment (LaurenceKuhl) [inline combinations/hash.tsv:606]:
```suggestion
r-base,r-igraph,r-dplyr,r-tidyr=1.3.0,r-readr=2.1.5,r-ggplot2=3.1.0
```

File: combinations/hash.tsv [hunk 2]
```diff
@@ -603,4 +603,4 @@ lumpy-sv=0.3.1	quay.io/bioconda/base-glibc-busybox-bash:3.1	3
 bwameth=0.2.7,bwa-mem2=2.2.1
 pasta=1.9.0,pigz=2.8
 longphase=1.7.3,htslib=1.20
-r-base,r-igraph,r-dplyr
+r-base,r-igraph,r-dplyr,r-tidyr=1.3.0,r-readr=2.1.5,r-ggplot2=3.1.0
```
Comment (LaurenceKuhl) [inline combinations/hash.tsv:606]:
```suggestion
r-base,r-igraph,r-dplyr,r-tidyr,r-readr,r-ggplot2
```

**Comment by LaurenceKuhl** [comment]
hi sorry asked for a review too early :(

File: combinations/hash.tsv (hunk 2 above)
Comment (LaurenceKuhl) [inline combinations/hash.tsv:606]:
```suggestion
r-base=4.4.1,r-igraph=2.0.3,r-dplyr=1.1.4,r-tidyr=1.3.1,r-readr=2.1.5,r-ggplot2=3.1.0
```

**Comment by LaurenceKuhl** [comment]
hi @bgruening i'm trying to figure out the versions so create locally a conda environment with all packages i needed and then manually checked which versions of my R packages were installed there and then appended them to this commit, so i'm not sure why it's failing in the container? Is there any other way for me to figure out matching versions? 
Thanks a lot for any link towards any doc in case i missed something :)

**Comment by bgruening** [comment]
I did the same than you did `conda create -n gasdf --dry-run r-base r-igraph r-dplyr r-tidyr r-readr r-ggplot2` and failed to write down the correct deps. As soon as I was able to write down what my eyes where seeing it worked :)

**Pull request merged**